src/PackageJsonSynchronizer.php

```diff
@@ -98,11 +100,8 @@ private function registerWebpackResources(array $phpPackages)
                 continue;
             }
 
-            if (!file_exists($packageJsonPath = $this->rootDir.'/vendor/'.$phpPackage.$assetsDir.'/package.json')) {
-                continue;
```

Comment:
Why removing this ?

Reply:
Because it's useless, we're sure the file exists if we reach this line.